rtt_ros2_node/src/rtt_ros2_node_service.cpp

```diff
@@ -74,7 +74,7 @@ static void loadGlobalROSService()
 
   // Call rclcpp::init()
   rclcpp::InitOptions init_options;
-  init_options.shutdown_on_sigint = false;
+  init_options.shutdown_on_signal = false;
```

Comment:
This flag was [renamed in version 13.1.0 of rclcpp](https://github.com/ros2/rclcpp/blob/13.1.0/rclcpp/CHANGELOG.rst#L9) (https://github.com/ros2/rclcpp/pull/1771), so only since ROS 2 Humble. Even though Galactic and older have reached their end-of-life already, I suggest to add a preprocessor condition in this case, to still support older ROS versions without having to maintain multiple branches.

Something like
```suggestion
#if RCLCPP_VERSION_GTE(13, 1, 0)
  init_options.shutdown_on_signal = false;
#else
  init_options.shutdown_on_sigint = false;
#endif
```
and
```cpp
#include "rclcpp/version.h"
```
above should do.